pkg/apis/core/validation/validation.go

```diff
@@ -1644,7 +1647,9 @@ var allowedPVCTemplateObjectMetaFields = map[string]bool{
 // PersistentVolumeSpecValidationOptions contains the different settings for PeristentVolume validation
 type PersistentVolumeSpecValidationOptions struct {
 	// Allow spec to contain the "ReadWiteOncePod" access mode
-	AllowReadWriteOncePod bool
+	AllowReadWriteOncePod       bool
```

Comment:
I bet gofmt will want this line back the way it was

Reply:
@liggitt true :), Done.